nihruk/design-system
# Consolidate Brand and Identity and Design into a new Styles section.

## How to test
- Navigate to https://design-system-git-styles-nihruk.vercel.app/style
- Test the style navigation in the secondary column on this page, as well as each individual style's page.
- Confirm the navigation list looks like it must as part of the DS
